live/build-livecd-rfs

```diff
@@ -46,7 +46,11 @@ which mosb || {
   exit 1
 }
 choose_port() {
-  echo 59111 # TODO auto-select a free port
+	port=59000
+	while nc -z 127.0.0.1 $port; do
+		port=$((port + 1))
+	done
```

Comment:
I suggest randomizing the initial port selection, then parallel invocations wont try the same ports in sequence.

```
HOST_IP=127.0.0.1       
HOST_BASE=59000                            
HOST_PORT=$HOST_BASE                            
while nc -z "$HOST_IP" "$HOST_PORT"; do         
    HOST_PORT=$((HOST_BASE + (RANDOM % 1000)))  
done                                            
```

Reply:
Exactly what do you have in mind?  Just
```
BASE=$((20000 + RANDOM % 20000))
```
?

Reply:
(apologies - odd, somehow i hadn't noticed your inline suggestion when I saw this comment before!)